File: mobile/lib/features/chauffeur/domain/entities/geolocalisation.dart
/// Géolocalisation — miroir de [Geolocalisation] côté backend.
class Geolocalisation {
  final int? id;
  final double? latitude;
  final double? longitude;
  final DateTime? horodatage;

  const Geolocalisation({
    this.id,
    this.latitude,
    this.longitude,
    this.horodatage,
  });
}

File: backend/src/main/java/com/tmk/vtcmanager/application/domain/chauffeur/Geolocalisation.java
package com.tmk.vtcmanager.application.domain.chauffeur;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Geolocalisation {

    private Long id;
    private Double latitude;
    private Double longitude;
    private LocalDateTime horodatage;
}